src/components/menu-item/menu-item.jsx

```diff
@@ -7,11 +7,16 @@ const MenuContext = createContext();
 const useMenuContext = () => useContext( MenuContext );
 
 const Menu = ( { size = 'md', children, className } ) => {
-	const baseClasses = 'w-64 flex flex-col bg-background-primary p-2';
+	const baseClasses = 'flex flex-col bg-background-primary p-2';
+
+	const sizeClasses = {
+		sm: 'w-[200px]',
```

Comment:
Tailwind-Merge can't override Arbitrary classNames. You can define the width in the Tailwind config and use it here so that developers/Tailwind-Merge can easily override it.